services/navigation/navigation.go

```diff
@@ -64,11 +64,12 @@ func FromRobot(r robot.Robot, name string) (Service, error) {
 
 // Config describes how to configure the service.
 type Config struct {
-	Store              StoreConfig `json:"store"`
-	BaseName           string      `json:"base"`
-	MovementSensorName string      `json:"movement_sensor"`
-	DegPerSecDefault   float64     `json:"degs_per_sec"`
-	MMPerSecDefault    float64     `json:"mm_per_sec"`
+	Store               StoreConfig `json:"store"`
+	BaseName            string      `json:"base_name"`
+	MovementSensorName  string      `json:"movement_sensor_name"`
+	MotionServiceName   string      `json:"motion_service_name"`
+	DegPerSecDefault    float64     `json:"degs_per_sec"`
+	MetersPerSecDefault float64     `json:"meters_per_sec"`
```

Comment:
as requested: Hide functionality to add obstacles to the nav config, we see that we do not have 
```
`json:"obstacles"`
```